corehq/util/archive_old_forms.py

```diff
@@ -0,0 +1,89 @@
+from corehq.apps.es import cases as case_es
+from corehq.apps.es.forms import FormES
+from corehq.form_processor.models.forms import XFormInstance
+import datetime
+from math import ceil
+import os
+
+# Note: There are over 2.3 million cases that need to be archived on the target project. For this reason,
+# batching will be necessary. Furthermore, the batch sizes are kept small as each case can have 2-3+ forms
+# that will need to be archived.
+
+# Constants
+DOMAIN = 'test-65' # 'fmoh-echis-staging'
+DATE_CONSTRAINT = datetime.date(2022, 5, 1)
+BATCH_SIZE = 100
+ERROR_LOG_FILE_PATH = os.path.expanduser("~/zeng_script_error.log")
+SUCCESS_LOG_FILE_PATH = os.path.expanduser("~/zeng/script_success.log")
```

Comment:
Are you intentionally putting this file in a subdirectory, and the other log file in the home directory?

Reply:
That seems to be a missing slash, they should both be going to the `zeng` subdirectory.